io-sim/src/Control/Monad/IOSim/Types.hs

```diff
@@ -639,6 +640,13 @@ data SimResult a
 
 type SimTrace a = Trace.Trace (SimResult a) SimEvent
 
+traceEvents :: SimTrace a -> [(Time, ThreadId, Maybe ThreadLabel, SimEventType)]
```

Comment:
Do we need to move `traceEvents` to this module?

Reply:
No, that's something I forgot I had done, I'll revert this!

Reply:
Done